.gitignore

```diff
@@ -7,9 +7,9 @@
 /MacOS/build/Release/EnigmaXcode.app/Contents/Resources/*
 /definitions.h
 /coverage_*.info
-/libEGM.*
-/libProtocols.*
-
+/*EGM*.*
```

Comment:
part of it is cmake names things different by default but also it supports building static and dynamic versions as well as debug and release versions all in same dir so rather than enforcing that the names match our makefiles under all conditions I just had it ignore all variations of names